### ISSUE TITLE
requirejs config wrong in 4.1.0 - tries to load typeahead from wrong location

### ISSUE THREAD

**Issue by jasongrout** [jupyter/notebook#1154]
The requirejs config in the notebook page template in 4.1.0 is wrong, and tries to load the jquery-typeahead dependency from the wrong location. The line https://github.com/jupyter/notebook/blob/4.1.0/notebook/templates/page.html#L41 should be changed from:

```
            typeahead: 'components/jquery-typeahead/dist/jquery.typeahead'
```

to

```
            typeahead: 'components/jquery-typeahead/dist/jquery.typeahead.min'
```

(i.e., add `.min` to the end of the path)

As a result, `jupyter notebook --NotebookApp.ignore_minified_js=True` hangs when showing a notebook and just shows a blank screen.

**Comment by takluyver** [comment]
@jasongrout is this still an issue? Can you turn your fix into a PR?

**Comment by minrk** [comment]
Fixed by #1917
